spec/models/move_spec.rb

```diff
@@ -256,12 +256,37 @@
   end
 
   describe '#move_type' do
-    subject(:move) { build :move, from_location: from_location, to_location: to_location, move_type: nil }
+    subject(:move) { build :move, from_location: from_location, to_location: to_location, move_type: move_type, version: version }
 
     let(:from_location) { build :location, :police }
+    let(:move_type) { nil }
+    let(:version) { nil }
 
     before { move.valid? }
 
+    context 'when creating a v2 move' do
```

Comment:
At some point this will become the default behaviour (i.e. when v1 support is removed)